### ISSUE TITLE
gama stays on in mac sierra activity monitor 

### ISSUE THREAD

**Issue by sriramab** [gama-platform/gama#2083]
A participant yesterday in the course had installed gama 1.6, gama 1.7 and also latest builds. Strangely even if we close or force quit gama, gama stays in activity monitor.

secondly, java 8 installed but mac does not show 1.8 installed. the participant had only apple java 1.6 and he could not do any gama assignments :( 

Any ideas to fix this? so that he can work at least today?

**Comment by sriramab** [comment]
He is using macbook Air, if this makes any difference

**Comment by AlexisDrogoul** [comment]
There are several causes that need to be sorted here before we can make repeatable tests: which version of gama used to stick ? would killing the process (from the activity monitor) leave it untouched also ? How does the participant know that java 1.8 was installed ? (this problem of Java version not found is not specific to GAMA, see for instance https://www.eclipse.org/forums/index.php/t/1068247/ or http://stackoverflow.com/questions/35246910/eclipse-not-recognizing-jvm-1-8)

**Comment by sriramab** [comment]
1. First we installed latest build, while user had a 1.6 working copy. So that he can follow along the assignments of the course. I assume 1.6 was working because it connects to apple java 1.6
2. Before  installing latest build, we had to install java 1.8. We did. We ran latest build, it did not open. It gave an error log somewhere into eclipse.app/contents/... and from what I understand it was having errors with xtext
3. I gave him my working copy of somewhere from early december 2016, also same error as point 2 above. 
4. we restarted mac machine. But error continued with all three, 1.6 also collapsed. 
5. we checked on terminal using java -version, it showed only apple java even after we installed 1.8
6. we opened activity monitor and there was gama 1.6, gama 1.7 latest build .. we could recognise this because they have different icons. But surprisingly we could not quit them as they were grayed out. Also little down the list in activity monitor java 1.8 was also grayed out. 

7. we uninstalled java 1.8 from stackoverflow suggested methods and we install java1.8.112. No luck. I had to stop as it took almost 20 minutes and I had to continue with the course with remaining participants.
